### ISSUE TITLE
Feature request: ability to change network settings without bluetooth.

### ISSUE THREAD

**Issue by millerz1897** [NebraLtd/helium-miner-software#218]
Having difficulty (all versions including latest 11.11 update) connecting and maintaining wireless connectivity via helium app ( last 3 versions including the moat recent 3.8.0). Would be very nice to have an administrative interface over hardwired Ethernet to adjust, reboot, etc.

**Comment by shawaj** [comment]
@millerz1897 we are actually planning that but it's blocked by https://github.com/NebraLtd/hm-diag/issues/39 currently

**Comment by millerz1897** [comment]
> @millerz1897 we are actually planning that but it's blocked by [NebraLtd/hm-diag#39](https://github.com/NebraLtd/hm-diag/issues/39) currently

Thanks, to be clear the reference points to an insecure scenario where a we interface is exposed to the internet. (Lte module, poor configurations, etc.  One solution would be a dashboard.  But that wouldn't help fix network config issue.

so, to be more clear an interface, only accessible via ethernet, with only the ability to have enhanced diagnoatic info and the ability to change adapter setting, is the recomendation.

**Comment by shawaj** [comment]
@millerz1897 understand, but local internet is always possible to accidentally expose to the public internet.

So sensitive things like WiFi need to be needing authentication
